Gems/SaveData/Code/Source/Platform/Windows/SaveData_SystemComponent_Windows.cpp

```diff
@@ -62,7 +62,7 @@ namespace SaveData
 
         ////////////////////////////////////////////////////////////////////////////////////////////
         //! The absolute path to the application's save data dircetory.
-        AZStd::string m_saveDataDircetoryPathAbsolute = nullptr;
+        AZStd::string m_saveDataDircetoryPathAbsolute;
```

Comment:
@bosnichd I am bringing this to your attention, as you may need to implement this change to the SaveData gem for other files as well in order to keep on compiling

Reply:
Thanks for the heads up! I just checked and you already covered all the relevant bases in this review.

This must have been a const char* originally before being changed to a string and forgetting to remove the '= nullptr;' - I'm actually pretty surprised it was ever allowed to init a string class with nullptr, so I think this is a great change!